Automattic/wp-calypso
# Guided Onboarding: Start the experiment

Related to 7021-gh-Automattic/dotcom-forge

## Proposed Changes

This PR:
- Removes the Cft structure
- Changes the flow name from 'guided' to 'onboarding'
- Checks for the experiment and uses the assignment
- Adds `trailmap_variant` to the event

## Why are these changes being made?


Starts the Guided Onboarding experiment! pbxNRc-3JR-p2

## Testing Instructions



- Go to the experiment page ( 21809-explat-experiment ) and assign yourself the appropriate group
- Test the flow
- Check that the experiment assignment is added to the page_view event
